composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Interop\\Async\\Loop\\": "test"
+            "Interop\\Async\\Loop\\Test\\": "test"
```

Comment:
That's not true. It only needs to access Loop class, which anyway needs to be imported.